multimedia/v4l2camera/patches/001_live555_sprintf_format

```diff
@@ -0,0 +1,25 @@
+From: Michel Promonet <michel.promonet@free.fr>
```

Comment:
this patch file lacks a proper suffix: 001_live555_sprintf_format<strong>.patch</strong>

Reply:
renamed